packages/react-scripts/fixtures/kitchensink/src/features/webpack/CssModulesInclusion.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+import React from 'react';
+import styles from './assets/style.module.css';
+
+export default () => (
+  <p className={styles.cssModuleInclusion}>We love useless text.</p>
```

Comment:
I attempted to add e2e but got stuck at [`const doc = await initDOM('css-modules-inclusion');`](https://github.com/ro-savage/create-react-app/blob/ece8be01caa7cc41f09deca785c9dd11e0dc105f/packages/react-scripts/fixtures/kitchensink/integration/webpack.test.js#L24).

I must be missing something fundamental because I could not get to load the correct component or match in the [`switch statement`](https://github.com/ro-savage/create-react-app/blob/ece8be01caa7cc41f09deca785c9dd11e0dc105f/packages/react-scripts/fixtures/kitchensink/src/App.js#L85)

Even when I updated the regular css-inclusions import statement to bring in the CSS Module component, it was still just loading the regular css component.

There must be some other way the components are being loaded for the e2e tests outside of `integration/webpack.test.js`